ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HttpTimeoutTest.kt

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.tests
+
+import io.ktor.client.call.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.client.tests.utils.*
+import io.ktor.http.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.core.*
+import kotlin.test.*
+
+class HttpTimeoutTest : ClientLoader() {
+    @Test
+    fun getTest() = clientTests {
```

Comment:
`testGet`

Reply:
`val TEST_URL="$TEST_SERVER/timeout/"`

Reply:
Consider adding tests with
 -  the response `cancel`
 - `withTimeout`
 - the empty body

Reply:
Added, please have a look.